extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ServerLimitsConfig.java

```diff
@@ -41,6 +41,27 @@ public class ServerLimitsConfig {
     @ConfigItem(defaultValue = "2048")
     public MemorySize maxFormAttributeSize;
 
+    /**
+     * Set the maximum number of fields of a form. Set to {@code -1} to allow unlimited number of attributes.
+     */
+    @ConfigItem(defaultValue = "256")
+    public int maxFormFields;
+
+    /**
+     * Set the maximum number of bytes a server can buffer when decoding a form.
+     * Set to {@code -1} to allow unlimited length
+     **/
+    @ConfigItem(defaultValue = "1K")
+    public MemorySize maxFormBufferedBytes;
+
+    /**
+     * The maximum number of HTTP request parameters permitted for incoming requests.
+     * <p>
+     * If a client sends more than this number of parameters in a request, the connection is closed.
+     */
+    @ConfigItem(defaultValue = "1000")
+    public int maxParameters;
```

Comment:
@jponge  this config parameter doesn't seem to be used

Reply:
You're right, it might be removed